quizbizz/src/pages/Homepage/AcheivementCategorySection/Categories/Categories.jsx

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import "./categories.css";
+import CategoryCard from "./CategoryCard";
+import JavaScriptImage from "../../../../assets/images/javascript.png";
+import CssImage from "../../../../assets/images/css.png";
+import JavaImage from "../../../../assets/images/java.jpeg";
+import ReactImage from "../../../../assets/images/react.png";
```

Comment:
Common export file can be made for images

Reply:
Done